docs/api/lights/DirectionalLightShadow.html

```diff
@@ -12,7 +12,7 @@
 
 		<h1>[name]</h1>
 
-		<div class="desc">TODO</div>
+		<div class="desc">Directional light with a shadow map, for casting shadows.</div>

```

Comment:
... and this is why one should never accept pull requests that have "TODO" in the documentation. I'll go fix it and add to the PR.